extensions/framer-motion/src/utilities.tsx

```diff
@@ -0,0 +1,26 @@
+import { Action, ActionPanel, List } from "@raycast/api";
+
+import { UTILITY_ITEMS } from "./data";
+
+export default function Command() {
+  return (
+    <List>
+      {UTILITY_ITEMS.map((items) => (
+        <List.Section key={items.section} title={items.section}>
+          {items.items.map((item) => (
+            <List.Item
+              key={item.title + Math.random()}
+              title={item.title}
+              subtitle={item.subtitle}
```

Comment:
Adding the section title as a keyword for each item could be useful if you're not sure what the utility is named e.g. searching for just "3d" or "transition".

![Sam 2024-03-22 at 10  23 49@2x](https://github.com/raycast/extensions/assets/2223418/4128fbbd-cd55-4b49-b7a4-b6bc4547230d)



```suggestion
              subtitle={item.subtitle}
              keywords={[items.section]}
```